src/plugins/discover/public/application/components/visualizations/vislib/line/line_vis_type.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { i18n } from '@osd/i18n';
+
+import { VisualizationType } from '../../../../view_components/utils/use_visualization_type';
+import { toExpression } from './to_expression';
+
+export interface LineOptionsDefaults {
+  type: 'line';
+}
+
+export const createLineConfig = (): VisualizationType<LineOptionsDefaults> => ({
```

Comment:
we have create_histogram_configs  chart/utils/ would it make more sense to add it to that folder?

Reply:
sure good idea